### DISCUSSION TITLE
Pronouncing trurl

### DISCUSSION THREAD

**Discussion by bagder** [curl/trurl#62] [Show and tell]
Pretend there is an 'e' between the r and the l: "trurel"

Here is a short [mp3 audio clip](https://curl.se/trurl/trurl.mp3) with me saying it.

**Comment by dfandrich** [comment]
We pendants want the IPA! Do you mean [ˈtɹʊɹəɫ](http://ipa-reader.xyz/?text=%CB%88t%C9%B9%CA%8A%C9%B9%C9%99%C9%AB) (like plural), [ˈtɹuɹəɫ](http://ipa-reader.xyz/?text=%CB%88t%C9%B9u%C9%B9%C9%99%C9%AB) (longer u sound),  [ˈtɹəɹəɫ](http://ipa-reader.xyz/?text=%CB%88t%C9%B9%C9%99%C9%B9%C9%99%C9%AB) (shorter u), [ˈtɹuəɫ](http://ipa-reader.xyz/?text=%CB%88t%C9%B9u%C9%99%C9%AB) (more like cruel), or something else?

I think you need to record a short audio clip (in raw 8k Sun audio format) saying something like *Hello, this is Daniel Stenberg and I pronounce trurl as "trurl"*.

**Reply by bagder** [reply]
I think maybe [ˈtɹuɹəɫ](http://ipa-reader.xyz/?text=%CB%88t%C9%B9u%C9%B9%C9%99%C9%AB) matches my own pronunciation the most. I'll try to make a recording later.

**Reply by bagder** [reply]
I added a link to the original post above.
